intranet/apps/polls/models.py

```diff
@@ -94,6 +94,23 @@ def get_users_voted(self):
                 users = list(q.get_users_voted())
         return users
 
+    def get_num_eligible_voters(self):
+        users = []
+        if self.groups.all():
+            for g in self.groups.all():
+                if users:
+                    users = list(set(DjangoGroup.objects.get(name=g).user_set.all()) | set(users))
```

Comment:
Similar behavior to this can be achieved with Django's `union`. That also eliminates the need for this `if`-`else` block.